js/screens/Profile/styles.js

```diff
@@ -1,2 +1,74 @@
-import { StyleSheet } from "react-native";
-export default (style = StyleSheet.create({}));
+import { StyleSheet, Dimensions } from "react-native";
+import globalStyles from "../../config/styles";
+const { height, width } = Dimensions.get("window");
+
+const styles = StyleSheet.create({
+  profileContainer: {
+    height,
+    width,
+    alignItems: "center",
+    justifyContent: "space-around"
+  },
+  accountName: {
+    // fontFamily: globalStyles.titleFont.fontFamily,
```

Comment:
Remove comments.

Reply:
the font families have not been added to the project yet. as soon as they are these lines will be uncommented so to apply the correct font.